### PR TITLE
M1 installation documentation

### PR DESCRIPTION
**Pull request by talonchandler**
Interim fix for #154.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,3 +85,5 @@ python
 ```
 
 To use GPUs in `waveorder` set ```use_gpu=True``` when initializing the simulator and reconstructor classes.
```
Comment (ziw-liu) [inline README.md:87]: This is the CuPy toggle and not relevant to the PyTorch implementation.

Comment (ziw-liu) [reply README.md:87]: I don't know if it still works or not, but we should consider removing any mention of GPU in user-facing documentation before we have a fully working torch port.

Comment (talonchandler) [reply README.md]: I agree, I just removed this section from the README.

File: README.md [hunk 2]
```diff
@@ -77,11 +77,4 @@ pip install jupyter
...
 
-(Optional) Use NVIDIA GPUs by installing `cupy` with [these instructions](https://docs.cupy.dev/en/stable/install.html). Check that `cupy` is properly installed with
-
-```sh
-python
->>> import cupy
-```
-
-To use GPUs in `waveorder` set ```use_gpu=True``` when initializing the simulator and reconstructor classes.
+(M1 users) `pytorch` has [incomplete GPU support](https://github.com/pytorch/pytorch/issues/77764), so please use `export PYTORCH_ENABLE_MPS_FALLBACK=1` to fallback to CPU.
```
Comment (ziw-liu) [inline README.md:80]:
```suggestion
(M1 users) `pytorch` has [incomplete GPU support](https://github.com/pytorch/pytorch/issues/77764),
so please use `export PYTORCH_ENABLE_MPS_FALLBACK=1`
to allow some operators to fallback to CPU
if you plan to use GPU acceleration for polarization reconstruction. 
```

**Review by ziw-liu — APPROVED** [review]

**Pull request merged**